tests/torch/requirements.txt

```diff
@@ -24,3 +24,4 @@ timm==0.9.2
 # Required for torch/fx tests
 torchvision
 fastdownload==0.0.7
+ultralytics==8.2.56  # TODO(dlyakhov) move ultralytics requirements to the nightly test
```

Comment:
Could you think about how to avoid adding `ultralytics` as dependency for pre-commit tests.

Reply:
Done in latest merge from @daniil-lyakhov.